external-plugins/testutils/fakegithub.go

```diff
@@ -143,6 +143,21 @@ func (foc *FakeOwnersClient) ParseFullConfig(path string) (repoowners.FullConfig
 	return *full, err
 }
 
+func (foc *FakeOwnersClient) AllOwners() sets.Set[string] {
```

Comment:
Why are these required?

Reply:
The interface has been updated, therefore I needed to implement some more funcs.